TimelyQuotes/app/src/main/java/com/example/timelyquotes/QuoteGenerator.java

```diff
@@ -0,0 +1,66 @@
+package com.example.timelyquotes;
+
+import java.util.ArrayList;
+
+public class QuoteGenerator {
+
+    // Titles of the notifications to be displayed
+    // structure: [artist name], "[song name]", [year]
+    private static ArrayList<String> titles = new ArrayList<>();
```

Comment:
Why using a static string list of both the titles and the quotes instead of using OOP and creating a "Quote" class?